desktop/desktop.scss

```diff
@@ -21,9 +31,10 @@
   #main-container + .reviewable,
   #main-container + .discourse-post-event-upcoming-events,
   #main-container + div {
-    grid-area: 1;
+    grid-row-start: navigation;
+    grid-row-end: content;
   }
-  &.category:not(.forza-hero-home) #main-outlet {
+  &.category:not(.forza-hero-home):not(.categories-list) #main-outlet {
```

Comment:
Not new in this PR, but we shouldn't be using `forza-hero-home` here (even if its a class we want to exclude).